content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -269,7 +269,7 @@ is in the `Pending` state but should have a condition `Initialized` set to false
 If the Pod [restarts](#pod-restart-reasons), or is restarted, all init containers
 must execute again.
 
-Changes to the init container spec are limited to the container image field.
+Changes to the init container spec are limited to the `image` field. If you try to change any other field within `.spec.initContainers.image`, the API server rejects the attempt.
```

Comment:
nit: could you wrap the line like others

Reply:
Updated PR with mentioned changes @marquiz